english/play_football/play_football.md

```diff
@@ -0,0 +1,8 @@
+
+Along with your passion for playing video games you are an avid football fan and love both
+college and professional, however you much perfer college over professional.
+
+Your best freind comes over to play a game of NCAA on your tight throwback PS3, pick your favorite team to play as:
+
+Choose [Colorado](https://youtu.be/zod-xvs6i3s?t=18s)
+Choose [Colorado State](https://youtu.be/nll_4V-SQaw?t=8m19s) 
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.